<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1065</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1065</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="UI" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>301</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QPushButton" name="book_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
     </font>
    </property>
    <property name="text">
     <string>book</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sumbit_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
     </font>
    </property>
    <property name="text">
     <string>sumbit</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="query_btn">
       <property name="font">
        <font>
         <family>Algerian</family>
        </font>
       </property>
       <property name="text">
        <string>query</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="id_label_2">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="id_edit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="log_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>269</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>147</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="curent_time_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>current time：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_time_display">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>第 1 天 0 点</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="enable_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
     </font>
    </property>
    <property name="text">
     <string>en</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="id_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="id_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="strategy_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>strategy：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="destination_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>destination：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="time_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>time limit：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="strategy_box">
       <property name="currentText">
        <string>最小风险</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>最小风险</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>限时最小风险</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="destination_edit"/>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="time_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="departure_label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
        </font>
       </property>
       <property name="text">
        <string>departure：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="departure_edit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
